src/main/java/nl/mtvehicles/core/events/JerryCanClickEvent.java

```diff
@@ -0,0 +1,14 @@
+package nl.mtvehicles.core.events;
+
+import nl.mtvehicles.core.infrastructure.annotations.ToDo;
+import nl.mtvehicles.core.infrastructure.models.MTVEvent;
+import org.bukkit.event.Cancellable;
+
+@ToDo(comment = "More methods.")
+public class JerryCanClickEvent extends MTVEvent implements Cancellable {
```

Comment:
Maybe a method where you can get information about the jerry can

Reply:
Added!